### PR TITLE
Add data ingestion call

### PR DESCRIPTION
**Pull request by thebriz24**
## Brief context on code (tell us why these changes are necessary)
Add a call for posting data to our data ingestion endpoint.

## Test plan
Use `ExKumete.Session.start` with the client id and secret of your choice. Then call `ExKumete.DataIngestion.create` with the token of this first call, and a map that has a data field, a integration name, and a transform uid.

## Before asking for code review

- [x] I have unit tested behavioral changes
- [x] I have verified test plan works
- [x] I have merged the base branch into this branch
- [x] I have ensured that CI is passing
- [x] I have filled out the "Brief context on code" and "Test plan" sections above
- [ ] I have linked to the Jira ticket in the description above

### CODE REVIEW

File: .idea/ex_kumete.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="RUBY_MODULE" version="4">
```
Comment (matelau) [inline .idea/ex_kumete.iml]: this looks like it needs a git rm not sure why it didn't ignore 🤷‍♂️

Comment (thebriz24) [reply .idea/ex_kumete.iml]: Yeah I don't know why it doesn't listen...

File: lib/data_ingestion.ex [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  @spec create(token, map()) :: {atom, status_code, return}
+  def create(token, params) do
+    Logger.info("Sending event: [#{inspect(params)}]")
+
+    case Poison.encode(params) do
+      {:ok, encoded} ->
+        @data_path
+        |> Http.post(encoded, request_headers(token))
+        |> Rest.handle_response()
```
Comment (matelau) [inline lib/data_ingestion.ex]: just adding a note here that we will want to return camel cases keys

Comment (thebriz24) [reply lib/data_ingestion.ex]: Kumete will be responsible for the camel case response; I just updated it.

**Pull request merged**